docs/dataproc.md

```diff
@@ -0,0 +1,99 @@
+## Integration with DataProc / PySpark
+
+Bigflow provides integration with Pyspark running on Dataproc.
+It allows to adapt any python callable/function with PySpark code into `dataflow` Job.
+
+### Installation
+
+```shell
+pip install bigflow[dataproc]
+```
+
+### Define basic job
+
+Each PySpark job must have an entry point: python callable, passed as `driver` parameter. 
```

Comment:
Python callable (for example, a function)

Reply:
done